recipes/fibertools-rs/build.sh

```diff
@@ -17,6 +16,8 @@ export DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}:${LIBTORCH}/lib
 #
 # download pytorch libraries
 #
+if [[ "x" == "y" ]]; then 
```

Comment:
What is the idea here ? To disable the body of the `if` ?
IMO it would be better to either remove the body completely if it is not needed, or at least add a comment to explain why it is disabled and when it might be re-enabled.